### PR TITLE
refactor(core/best-practices): replace jQuery

### PR DESCRIPTION
**Pull request by saschanaz**
Because hyperHTML is great 😆

### CODE REVIEW

File: tools/release.js [hunk 1]
```diff
@@ -395,7 +395,7 @@ const run = async () => {
       colors.info(" Making sure the generated version is ok... 🕵🏻")
     );
     await node(
-      `./tools/respec2html.js -e --timeout 30 --src file://${process.cwd()}/examples/basic.built.html --out /dev/null`,
+      `./tools/respec2html.js -e --timeout 30 --src file:///${__dirname}/../basic.built.html --out /dev/null`,
```
Comment (saschanaz) [inline tools/release.js]: I accidentally missed `../` part, but still think `__dirname` is better because it's cwd-safe. What do you think? @marcoscaceres

**Comment by saschanaz** [comment]
Codacy doesn't like the unused `conf` but it's currently needed for base-runner to detect modern modules...

File: tools/release.js (hunk 1 above)
Comment (marcoscaceres) [reply tools/release.js]: I'm ok with __dirname, but it was breaking. I usually run the release script from:

```Bash
./tools/release.js
``` 

Thus need to make sure that it always resolves to the right place.

File: src/core/aria.js [hunk 2]
```diff
@@ -10,7 +10,6 @@ sub(
   { once: true }
 );
 
-export function run(conf, doc, cb) {
-  doc.body.setAttribute("aria-busy", "true");
-  cb();
+export function run(conf) {
+  document.body.setAttribute("aria-busy", "true");
```
Comment (marcoscaceres) [inline src/core/aria.js]: Oh, I thought I'd removed this module? This module is no good. Could you delete it?

File: src/core/best-practices.js [hunk 3]
```diff
@@ -4,34 +4,33 @@
...
-    $bp.prepend(doc.createTextNode("Best Practice " + num + ": "));
-  });
-  if ($bps.length) {
-    $(doc).find("head link").first().before($("<style/>").text(css));
-    if ($("#bp-summary")) $("#bp-summary").append($content.contents());
-  } else if ($("#bp-summary").length) {
+    const id = window.$.fn.makeID.call(bp, "bp");
+    const li = hyperHTML`<li><a href="${`#${id}`}">Best Practice ${num}</a>: ${bp.textContent}</li>`;
+    ul.appendChild(li);
+    bp.insertBefore(document.createTextNode(`Best Practice ${num}: `), bp.firstChild);
```
Comment (marcoscaceres) [inline src/core/best-practices.js]: still not a `wire()` fan, huh? 😉 That's ok.

Comment (saschanaz) [reply src/core/best-practices.js]:
```js
const num = 1;
hyperHTML`Best Practice ${num}:`
```

This returns 4 nodes. Inserting multiple nodes with `insertBefore` is kind of pain :/

Comment (marcoscaceres) [reply src/core/best-practices.js]: Yeah, that makes sense... good point.

**Comment by saschanaz** [comment]
No idea why the test fails, it passes on my machine. I should investigate more...

**Comment by saschanaz** [comment]
I had wrong configuration 😱 Now fixed. Unfortunately the Codacy warnings should be ignored for now...

**Pull request merged**